### PR TITLE
OSDOCS#6503: IBM Cloud VPC - move install config parameters out of as…

### PR DESCRIPTION
**Pull request by mjpytlak**
Version(s):
4.14+

Issue:
This PR addresses [osdocs-6503](https://issues.redhat.com/browse/OSDOCS-6503)

Link to docs preview:
- [Installation configuration parameters for IBM Cloud VPC](https://63813--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installation-config-parameters-ibm-cloud-vpc)
  New assembly at book level.
- Installing a cluster on IBM Cloud VPC with customizations > [Creating the installation configuration file](https://63813--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-customizations#installation-initializing_installing-ibm-cloud-customizations)
Note the absence of Installation configuration parameters following this topic. In lieu of the module, a cross reference is provided for the new assembly.

It is worth noting that the latter is an example of the change that this PR introduces. This change has been applied to all IBM Cloud VPC installation assemblies.

QE review:
- [N/A] QE has approved this change.
Given that this update is a reorg only, QE approval is not required. See conversation at https://github.com/openshift/openshift-docs/pull/61165#pullrequestreview-1493117699

Additional information:

### CODE REVIEW

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -208,28 +191,31 @@ endif::[]
 ifeval::["{context}" == "installation-config-parameters-bare-metal"]
 :bare:
 endif::[]
+ifeval::["{context}" == "installation-config-parameters-ibm-cloud-vpc"]
+:ibm-cloud:
+endif::[]
```
Comment (mjpytlak) [inline modules/installation-configuration-parameters.adoc:196]: Worth noting that while the original implementation of this module also assigned the `ibm-cloud-vpc` attribute, I deliberately removed it. When I implemented that work, I do so for a single sentence related to deploying a cluster to an existing VPC. Removing it was easily accommodated here. [1]

[1]  https://github.com/openshift/openshift-docs/pull/63813/files#diff-14ebe576efb84b2f58b32a0de2996fced237204ae9b091e3cafbd40b65c1ef31R513

**Comment by sjstout** [comment]
LGTM!

**Comment by stevsmit** [comment]
LGTM

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -524,9 +510,9 @@ networking:
 |`networking.machineNetwork.cidr`
 |
 Required if you use `networking.machineNetwork`. An IP address block. The default value is `10.0.0.0/16` for all platforms other than libvirt and {ibmpowerProductName} Virtual Server. For libvirt, the default value is `192.168.126.0/24`. For {ibmpowerProductName} Virtual Server, the default value is `192.168.0.0/24`.
-ifdef::ibm-cloud-vpc[]
-The CIDR must contain the subnets defined in `platform.ibmcloud.controlPlaneSubnets` and `platform.ibmcloud.computeSubnets`.
-endif::ibm-cloud-vpc[]
+ifdef::ibm-cloud[]
+If you are deploying the cluster to an existing Virtual Private Cloud (VPC), the CIDR must contain the subnets defined in `platform.ibmcloud.controlPlaneSubnets` and `platform.ibmcloud.computeSubnets`.
+endif::ibm-cloud[]
```
Comment (mjpytlak) [inline modules/installation-configuration-parameters.adoc:515]: @gpei Thank you for looking at this. Very small change here. I removed the `ibm-cloud-vpc` attribute to accommodate this refactor. If you are curious about this effort, you can find more details here [1]

[1] https://issues.redhat.com/browse/OSDOCS-6493

Comment (gpei) [reply modules/installation-configuration-parameters.adoc:515]: @mjpytlak thanks, this change looks good to me

**Pull request merged**